ui/k-info-panel/switch_info.kytos

```diff
@@ -55,13 +63,42 @@
        this.get_flows()
      },
      get_flows() {
+       this.nextState = this.metadata.enabled == 'true'? 'Disable': 'Enable'
+       console.log(this.nextState)
        var dpid = this.metadata.dpid
        var endpoint = this.$kytos_server_api + "kytos/flow_manager/v2/flows/" + dpid
        var self = this
        window.d3.json(endpoint, function(error, result) {
          self.flows = result[dpid].flows
        })
-     }
+     },
+     msg_state_success(data){
+       let notification = {
+         title: 'Switch ' + this.nextState + 'd: Succeed',
+         description: 'The switch ' + this.metadata.dpid + ' was ' + this.nextState.toLowerCase() + 'd.',
+         icon: 'gear',
+       }
+       this.nextState = this.nextState == 'Enable'? 'Disable' : 'Enable'
+       this.$kytos.$emit("setNotification", notification)
+     },
+     msg_state_failure(data){
+       let notification = {
+         title: 'Switch ' + this.nextState + 'd: Failed',
+         description: data.status + ': ' + data.responseJSON.description + '. The switch ' + this.metadata.dpid + ' was not ' + this.nextState.toLowerCase() + 'd.',
+         icon: 'gear',
+       }
+       this.$kytos.$emit("setNotification", notification)
+     },
+     bt_stateToggle(){
+       console.log("Hola->", this.nextState)
```

Comment:
Same as the previous console.log comment.

Reply:
It is now deleted.

Reply:
It is now deleted. Commit [4e91ab3ee60976a1d6fabfb1cb041793c253b9ed](https://github.com/kytos-ng/topology/pull/70/commits/4e91ab3ee60976a1d6fabfb1cb041793c253b9ed)